test/integration/smoke/test_vpc_ipv6.py

```diff
@@ -0,0 +1,901 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
```

Comment:
@shwstppr does it make sense to move the test_network_ipv6 to smoke/ ? How much time do they take? (asking as the test_vpc_ipv6 is in smoke but the other one is in component/ )

Reply:
@rohityadavcloud There are four test files added,
`smoke/test_network_ipv6.py` - contains 1 test-case
`smoke/test_vpc_ipv6.py` - contains 1 test-case
`component/test_network_ipv6.py` - contains 4 test-cases for upgrade network, redundant VR, etc
`component/test_vpc_ipv6.py` - contains 4 test-cases for upgrade VPC, redundant VR, etc

Each component test file may take over 30mins to execute

Reply:
Thanks @shwstppr